.changeset/sweet-trains-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ts-rest/express': major
```

Comment:
If this is a backwards-compatible update, let's make this a minor update :)

Reply:
good point, fixed in new version.